guides/common/modules/con_installing-and-configuring-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,11 @@
+[id="installing-and-configuring-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment"]
+= Installing and configuring the {Team} Insights On-Premises advisor engine
+
+The {Team} Insights On-Premises (IoP) advisor engine helps {RHEL} systems perform efficiently, keeping them secure and available.
```

Comment:
For Foreman I think this should just read "The advisor engine" and for Satellite "The Insights in Satellite advisor engine"